### ISSUE TITLE
[Proposal] Add TorchRL to IsaacLab workflows

### ISSUE THREAD

**Issue by fyu-bdai** [isaac-sim/IsaacLab#1181]
### Proposal

[TorchRL ](https://pytorch.org/rl/stable/index.html) is a RL framework produced by the PyTorch team that is growing in popularity within the research community. It offers fast optimization times, flexibility, and leverages tensordicts for data passing between environment and algorithm. It would be nice to add TorchRL as a new workflow option to IsaacLab. 

### Motivation

Please outline the motivation for the proposal. Summarize the core use cases and user problems and needs you are trying to solve.

Is your feature request related to a problem? e.g.,"I'm always frustrated when [...]".

If this is related to another GitHub issue, please link here too.

### Alternatives

A clear and concise description of any alternative solutions or features you've considered, if any.

### Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)

### Acceptance Criteria

Add the criteria for which this task is considered **done**. If not known at issue creation time, you can add this once the issue is assigned.

- [ ] Complete TorchRL train and play pipeline integrated into IsaacLab.

**Comment by mpgussert** [comment]
Good suggestion! I'll pass it along :)

Thank you!

**Comment by fyu-bdai** [comment]
> Good suggestion! I'll pass it along :)
> 
> Thank you!

I've already created the PRs for this proposal :sweat_smile:

**Comment by T-K-233** [comment]
I wonder if there is any update on this. TorchRL is very nice to have.

**Comment by garylvov** [comment]
It seems to me that a wrapper on the torch RL side has superseded this?

I was looking at 

https://github.com/pytorch/rl/blob/0fb94112e5bdb41f3c6c6f5ba7807ec71d3a2230/examples/collectors/isaaclab_rnn_ppo_memory.py


and 

https://github.com/pytorch/rl/blob/83f3d50215aa8841dc79a3189ec6f052feb52c60/torchrl/envs/libs/isaac_lab.py#L11

Do you mind confirming @vmoens ? Thank you for your work on this!

**Comment by vmoens** [comment]
Yes, we have a new release with a good integration and validations scripts coming soon! 
We worked on the following features:
- better support for arbitrary reset and reset_to 
- better memory management + tutorial 
- optimizing collector throughput for Isaac-like envs 
- Optimized RNN-based policy execution through torch scan and triton kernels

Any missing feature please shout out!

**Comment by garylvov** [comment]
Awesome thank you! I'm just starting to play around with this I'll lyk if anything comes up :)
